src/components/HomeRedirect.tsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export const HomeRedirect = () => {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    navigate('/', { replace: true });
+  }, [navigate]);
+
+  return (<></>);
```

Comment:
For such case you can just `return null`. There is no sense to return a Fragment